### PR TITLE
Bump stripe from 10.10.0 to 10.11.0

### PR DESCRIPTION
**Pull request by filipefurtad0**
#### What? Why?

Bumps Stripe from 10.10.0 to 10.11.0

#### What should we test?

- green build

#### Release notes

Changelog Category (reviewers may add a label for the release notes):

- [ ] User facing changes
- [ ] API changes (V0, V1, DFC or Webhook)
- [x] Technical changes only
- [ ] Feature toggled

The title of the pull request will be included in the release notes.

### CODE REVIEW

**Comment by filipefurtad0** [comment]
Re-opening :slightly_smiling_face:

**Review by dacook — COMMENTED** [review]
👍 

I see that there's a `client_secret` stored in each cassette, which has changed. Is that because it's related to your testing account? (Last time it was generated on `"hostname":"gaetan-Dell-G15"`)
Sorry if we're going round in circles with masking secrets, I forget about this one..

**Comment by filipefurtad0** [comment]
> I see that there's a client_secret stored in each cassette, which has changed. Is that because it's related to your testing account? (Last time it was generated on "hostname":"gaetan-Dell-G15")
Sorry if we're going round in circles with masking secrets, I forget about this one..

Hum, no worries at all - well spotted. I think this relates to the machine sending the requests. Maybe we should mark the whole `X-Stripe-Client-User-Agent` section as hidden. What do you think?

File: spec/fixtures/vcr_cassettes/Stripe-v10.11.0/Spree_Gateway_StripeSCA/_credit/refunds_the_payment.yml [hunk 1]
```diff
@@ -303,20 +307,20 @@ http_interactions:
           "canceled_at": null,
           "cancellation_reason": null,
           "capture_method": "automatic",
-          "client_secret": "pi_3Ooefi4DenidF2LU1DyZ34Mm_secret_w379gPpBGChVwBEUepP0qlRB0",
+          "client_secret": "pi_3OqeRV3UJ5brpc2a0jgoatoG_secret_BSfmlMO7KoJyiterLxC48ygb5",
```
Comment (dacook) [inline spec/fixtures/vcr_cassettes/Stripe-v10.11.0/Spree_Gateway_StripeSCA/_credit/refunds_the_payment.yml:310]: The word "secret" raised a flag for me 😉 
Is this environmental (something you have stored in `.env.test.local`)? 

Perhaps it's coming from your test account, and it just shows that the test can run independently with either yours or Gaetan's account, which is great. If there's no risk posed by it being exposed, then we can probably just leave it.

**Review by dacook — COMMENTED** [review]
I was actually referring to "client_secret", which was clearly different between test runs. 

But the hostname deserves consideration too. I think it's fine to be exposed, because I don't think it's not a unique identifier that could be leveraged.. unless maybe there was a certain vulnerability in your OS.. hmm..
I think the user agent details shouldn't have a bearing on the test results, so we can probably strip them. Let's try that, do you think you'd be able to set that up?

I found an example here which looks appropriate: https://github.com/vcr/vcr/issues/201#issuecomment-14049851
Something like:
```
 i.request.headers.delete('X-Stripe-Client-User-Agent')
```

**Comment by mkllnk** [comment]
I can't see the Stripe bump, only cassettes.

**Comment by filipefurtad0** [comment]
> I can't see the Stripe bump, only cassettes.

Indeed. There is some confusion on my end (and maybe GH) coming from the same branch name as the one from dependabot... I could not manage to reconcile branches. So, I've opened [another PR here](https://github.com/openfoodfoundation/openfoodnetwork/pull/12252), I'll try to address any reviews there. Sorry for the confusion.

File: spec/fixtures/vcr_cassettes/Stripe-v10.11.0/Spree_Gateway_StripeSCA/_credit/refunds_the_payment.yml (hunk 1 above)
Comment (filipefurtad0) [reply spec/fixtures/vcr_cassettes/Stripe-v10.11.0/Spree_Gateway_StripeSCA/_credit/refunds_the_payment.yml]: Please see [this comment](https://github.com/openfoodfoundation/openfoodnetwork/pull/12252/commits/5653d542f696badbd30829d44945b152ab5a9d70#r1516342069).

**Comment by filipefurtad0** [comment]
Closed in favor of https://github.com/openfoodfoundation/openfoodnetwork/pull/12252.

**Pull request closed without merging**